pkg/gornir/gornir.go

```diff
@@ -16,6 +19,89 @@ type Gornir struct {
 	Logger    Logger     // Logger for the object
 }
 
+// NewGornir is a Gornir constructor.
+func NewGornir() *Gornir {
+	return new(Gornir)
+}
+
+// SetOption is a funcion that sets one or more options for a given Gornir.
+type SetOption func(r *Gornir) error
+
+// Build is a Gornir constructor with options.
+func Build(opts ...SetOption) (*Gornir, error) {
```

Comment:
I'd rename to `New` as it will read as `gornir.New()`

Reply:
Yeah, makes sense.